CONTRIBUTING.md

```diff
@@ -168,61 +184,82 @@ field should also be included. The fields `tfa` and `doc` can be completely remo
 
 The following is an example of a website that _does not_ support TFA:
 
-```YAML
-    - name: Netflix
-      url: https://www.netflix.com/us/
-      twitter: Netflixhelps
-      facebook: netflix
-      email_address: example@netflix.com (Only if available and monitored)
-      img: netflix.svg
-      lang: <ISO 639-1 language code> (Only for non-English websites)
+```JSON
+{
+  "Netflix": {
+    "domain": "netflix.com",
+    "url": "https://www.netflix.com/us/",
+    "contact": {
+      "facebook": "netflix",
+      "twitter": "Netflixhelps"
+    },
+    "keywords": [
+      "entertainment"
+    ]
+  }
+}
 ```
 
-The `lang` field is only used for non-English websites. The language codes should be lowercase [ISO 639-1][iso-lang-wikipedia] codes.
+The `language` field inside `contact` can be included for non-English websites. The language
+codes should be lowercase [ISO 639-1][iso-lang-wikipedia] codes.
 
 ### Exceptions & Restrictions
 
-If a site requires the user to do something out of the ordinary to set up 2FA or if 2FA is only available in specific countries, you can note this on the
-website.
-
-```YAML
- - name: Site Name
-   url: https://www.site.com/
-   img: site.svg
-   tfa:
-     - sms
-   exception: "Specific text goes here."
-   doc: <link to site TFA documentation>
+If a site requires the user to do something out of the ordinary to set up 2FA or if 2FA is
+only available in specific countries, you can note this on the website.
+
+```JSON
+{
+  "Site Name": {
+    "domain": "site.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "<link to site TFA documentation>",
+    "notes": "Specific text goes here.",
+    "keywords": [
+      "keyword"
+    ]
+  }
+}
 ```
 
 ### Adding a site that is only available in specific regions
 
-If a site (with or without 2FA) is only available in certain countries - for example a government site - you can note this with the `regions` field.
-
-```YAML
- - name: Site Name
-   url: https://www.site.com/
-   img: site.svg
-   tfa:
-     - totp
-   doc: <link to site TFA documentation>
-   regions:
-     - <ISO 3166-1 country code>
+If a site (with or without 2FA) is only available in certain countries - for example a
+government site - you can note this with the `regions` field.
+
+```JSON
+{
+  "Site Name": {
+    "domain": "site.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "<link to site TFA documentation>",
+    "keywords": [
+      "keyword"
+    ],
+    "regions": [
+      "us",
+      "ca"
+    ]
+  }
+}
 ```
 
 The country codes should be lowercase [ISO 3166-1][iso-country-wikipedia] codes.
 
```

Comment:
Should there be a mention of how to use the `additional-domains`, `recovery` and `custom-(hardware|software)` properties?

Perhaps something like below?



### Other Properties

- `additional-domains`
If a site exists at another domain in addition to the main domain that is listed in the `domain` field, you can mark this with the `additional-domains` property.
```JSON
{
  "Site Name": {
    "domain": "site.com",
    "additional-domains": [
      "site.net",
      "site.io"
    ],
    "tfa": [
      "totp"
    ],
    "documentation": "<link to site TFA documentation>",
    "keywords": [
      "keyword"
    ]
  }
}
```

- `recovery`
The recovery field can be used to link to acccount recovery documentation about what to do if you lose access to your 2FA method.
```JSON
{
  "Site Name": {
    "domain": "site.com",
    "tfa": [
      "totp"
    ],
    "documentation": "<link to site TFA documentation>",
    "recovery": "<link to site TFA recovery documentation>",
    "keywords": [
      "keyword"
    ]
  }
}
```
- `custom-software`/`custom-hardware`
If a site uses a proprietary software or hardware method, you can add specific details of what is being used. Examples would be Authy or non-U2F security keys
```JSON
{
  "Site Name": {
    "domain": "site.com",
    "tfa": [
      "custom-software",
      "custom-hardware"
    ],
    "custom-software": [
      "Authy"
    ],
    "custom-hardware": [
      "Yubico OTP"
    ],
    "documentation": "<link to site TFA documentation>",
    "keywords": [
      "keyword"
    ]
  }
}
```

Reply:
Right now I was just going to replace the outdated information (i.e. YAML -> JSON). But since you already did the hard work, I will graciously include it... :stuck_out_tongue_winking_eye: 

The Wiki is also in dire need of an update.